sharepoint-server-farm-ha/nic.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Ah, yes. I remember this issue from when we first tried to do Sharepoint long ago.